src/Movies.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+class Movies
```

Comment:
You should rename this class (and file) to be just Movie - because it only holds one movie at a time, not a collection of movies.

Reply:
all been amended to Movie